src/lib/y2partitioner/widgets/columns/size.rb

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/widgets/columns/base"
+
+module Y2Partitioner
+  module Widgets
+    module Columns
+      # Widget for displaying the `Size` column
+      class Size < Base
+        # @see Columns::Base#title
+        def title
+          # TRANSLATORS: table header, size of block device e.g. "8.00 GiB"
+          Right(_("Size"))
+        end
+
+        # @see Columns::Base#value_for
+        def value_for(device)
+          size = device_size(device)
+
+          return "" unless size
+
+          cell(size.to_human_string, sort_key(size.to_i.to_s))
```

Comment:
How would it work when you compare 1024 with 512, for instance?

Reply:
Apparently, it works fine. From the top of my head, this comes from the previous code, and it is done because the :sortKey should be an string. Anyway, I'll recheck it.

Reply:
> I'll recheck it.

Yes, it is as [it was](https://github.com/yast/yast-storage-ng/blob/2b033a9a3c16f4464e5b138b160171203babda05/src/lib/y2partitioner/widgets/blk_devices_table.rb#L172).

And the `sortKey` must be a string.

![error](https://user-images.githubusercontent.com/1691872/86613759-efb1b300-bfa9-11ea-9359-009a1d407470.png)